src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -3448,9 +3448,9 @@ exports[`Story Page snapshots should match snapshot for STY 1`] = `
 }
 
 .emotion-27 {
-  height: 0;
   overflow: hidden;
   position: relative;
+  height: 0;
```

Comment:
We could do with a tech ticket to deal with these full page snapshots I don't think they are valuable at all (I can create this)

Reply:
Actually now I think of it there is one already https://jira.dev.bbc.co.uk/browse/NEWSWORLDSERVICE-1770